src/partials/query.editor.html

```diff
@@ -9,16 +9,24 @@
     </div>
     <div>
       <div class="gf-form">
-        <gf-form-dropdown model="ctrl.target.target"
+        <label class="gf-form-label query-keyword width-9">Measurment</label>
+        <gf-form-dropdown class="width-16" model="ctrl.target.target"
           allow-custom="true"
           lookup-text="true"
           get-options="ctrl.getOptions($query)"
           on-change="ctrl.refresh()">
         </gf-form-dropdown>
       </div>
     </div>
-    <div class="gf-form gf-form--grow">
-      <div class="gf-form-label gf-form-label--grow"></div>
+  <div class="gf-form-inline"> 
```

Comment:
Indents?

Reply:
your right, i fixed it  with my next commit